packages/vscode-extension/lib/babel_transformer.js

```diff
@@ -97,7 +101,7 @@ function transformWrapper({ filename, src, ...rest }) {
     }
   }
 
-  return transform({ filename, src, ...rest });
+  return transform({ filename, src, options, plugins });
```

Comment:
why this change here? I actually fixed one issue in another transformer a while ago that was assuming only three elements in that object exists while `plugins` was added afterwards to the specification. Given we don't use all of them it'd be more future proof to have the ellipsis here

Reply:
Here's the PR: https://github.com/kristerkari/react-native-svg-transformer/pull/354/files